mods/battle-royale/mod_info.lua

```diff
@@ -5,11 +5,11 @@ name = "Battle Royale"
 
 -- version of the mod as seen in the mod manager and is used by the client to know
 -- what the latest version is
-version = 10
+version = 10.1
```

Comment:
And why does the major version of the mod increase every time?

Reply:
The vault system expects an integer value if I am not mistaken, we never used a major / minor versioning with mods.